framework/src/postprocessors/SideIntegralVariablePostprocessor.C

```diff
@@ -36,17 +40,19 @@ SideIntegralVariablePostprocessor::SideIntegralVariablePostprocessor(
     _fv(_fv_variable)
 {
   addMooseVariableDependency(&mooseVariableField());
+
+  _qp_integration = !_fv;
+}
+
+Real
+SideIntegralVariablePostprocessor::computeFaceInfoIntegral(const FaceInfo * const fi)
+{
+  return MetaPhysicL::raw_value(
+      (*_fv_variable)(makeCDFace(*fi, faceArgSubdomains(*_fv_variable, *fi))));
```

Comment:
this diffs, previous behavior was a getBoundaryFaceValue

this is great for internal values though